device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/buffers.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers.json.j2
```

Comment:
why SN2700 and not SN4700?

Reply:
on previous platforms (4410, 4700, 3700, 3800 ...) it was also linked directly to 2700 in the same way...
except 4600 and 4600c.
I will link it to 4700